tests/example/full.py

```diff
@@ -13,13 +16,16 @@
     ],
     'publisher': 'CERN',
     'publicationYear': '2015',
+    'resourceType': {
+        'resourceTypeGeneral': 'Dataset'
+    }
 }
 
 # Validate dictionary
-assert schema31.validate(data)
```

Comment:
Thanks for explanation. I am fine to keep it as is in this case. However we should be always careful with modification of existing test cases.